Firestore/core/src/firebase/firestore/remote/exponential_backoff.h

```diff
@@ -27,6 +27,8 @@ namespace firebase {
 namespace firestore {
 namespace remote {
 
+using Milliseconds = util::AsyncQueue::Milliseconds;
```

Comment:
Declaring this in a header like this creates a new alias for this type in the `remote` namespace for anyone that includes this header. We don't need more than one `Milliseconds` type. More generally, avoid the `using` keyword at the top level of headers altogether unless your intent really is to introduce an alias to be consumed throughout the code.

In this case, `util::AsyncQueue` already declares this type, so if you want to move this type to the top level you should do so in the `util` namespace, possibly near the declaration of `util::TimerId`.

Alternatively, just move this type alias declaration into the implementation file which keeps the clutter out of there without introducing a type throughout the codebase.